contrib/vmstat.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env drgn
+# Copyright (c) SUSE Linux.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""Dump /proc/vmstat statistics.
+
+   Skip the last enumerator item as it holds the number of enumrator values."""
+
+from drgn.helpers.linux.cpumask import for_each_online_cpu
+from drgn.helpers.linux.percpu import per_cpu
+
+
+def print_event_line(event, counter):
+    print(f"{event.name:<36} {counter.value_():>16}")
+
+
+print(f"{'Event':<36} {'Count':>16}")
+
+# 1) vm_zone_stat statistics
+print("VM_ZONE_STAT:")
+vm_zone_stat = prog["vm_zone_stat"]
+for event in prog.type("enum zone_stat_item").enumerators[:-1]:
+    print_event_line(event, vm_zone_stat[event.value].counter)
+print()
+
+# 2) vm_node_stat statistics are there since v5.14
+if "vm_numa_event" in prog:
+    print("VM_NODE_STAT:")
+    vm_node_stat = prog["vm_node_stat"]
+    for event in prog.type("enum node_stat_item").enumerators[:-1]:
+        print_event_line(event, vm_node_stat[event.value].counter)
+    print()
+
+# 3) vm_numa_event statistics are available only if CONFIG_NUMA is enabled
+if "node_subsys" in prog:
```

Comment:
This part fails on kernels without https://github.com/torvalds/linux/commit/f19298b9516c1a031b34b4147773457e3efe743b (in v5.14):

```
$ python3 -m vmtest.vm -k '5.10.*' python3 -Bm drgn ./contrib/vmstat.py
...
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/osandov/repos/drgn2/drgn/__main__.py", line 16, in <module>
    main()
  File "/home/osandov/repos/drgn2/drgn/internal/cli.py", line 209, in main
    runpy.run_path(script, init_globals=init_globals, run_name="__main__")
  File "<frozen runpy>", line 291, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "./contrib/vmstat.py", line 37, in <module>
    vm_numa_event = prog["vm_numa_event"]
                    ~~~~^^^^^^^^^^^^^^^^^
KeyError: 'vm_numa_event'
```

Maybe you meant to do something like this?

```suggestion
# 3) vm_numa_event statistics are there since v5.14. They are only populated if
# CONFIG_NUMA is enabled.
if "node_subsys" in prog and "vm_numa_event" in prog:
```